server/src/test/java/io/spine/examples/shareaware/server/market/MarketProcessTest.java

```diff
@@ -0,0 +1,139 @@
+/*
+ * Copyright 2023, TeamDev. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Redistribution and use in source and/or binary forms, with or without
+ * modification, must retain the above copyright notice and the following
+ * disclaimer.
+ *
+ * THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS
+ * "AS IS" AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT
+ * LIMITED TO, THE IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR
+ * A PARTICULAR PURPOSE ARE DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT
+ * OWNER OR CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL,
+ * SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT
+ * LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE,
+ * DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY
+ * THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT
+ * (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
+ * OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+ */
+
+package io.spine.examples.shareaware.server.market;
+
+import io.spine.examples.shareaware.market.Market;
+import io.spine.examples.shareaware.market.command.CloseMarket;
+import io.spine.examples.shareaware.market.command.ObtainShares;
+import io.spine.examples.shareaware.market.command.OpenMarket;
+import io.spine.examples.shareaware.market.command.SellSharesOnMarket;
+import io.spine.examples.shareaware.market.event.MarketClosed;
+import io.spine.examples.shareaware.market.event.MarketOpened;
+import io.spine.examples.shareaware.market.rejection.Rejections.SharesCannotBeSoldOnMarket;
+import io.spine.examples.shareaware.market.rejection.Rejections.SharesCannotBeObtained;
+import io.spine.examples.shareaware.server.market.given.MarketTestContext;
+import io.spine.server.BoundedContextBuilder;
+import io.spine.testing.server.blackbox.ContextAwareTest;
+import io.spine.testing.server.model.ModelTests;
+import org.junit.jupiter.api.AfterAll;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+import static io.spine.examples.shareaware.server.market.given.MarketTestEnv.*;
+
+@DisplayName("`Market` should")
+public final class MarketProcessTest extends ContextAwareTest {
+
+    @BeforeAll
+    static void beforeAll() {
```

Comment:
Maybe we could pull these `@BeforeAll` and `@AfterAll` up to the parent type? We keep repeating these over and over, in every test class.

Please review.